newdle/client/src/components/creation/creation.module.scss

```
@@ -81,10 +81,16 @@
   }
 
   .options {
-    display: flex;
     justify-content: space-between;
     margin: 1em 0 0 1em;
     color: #808080;
     padding-bottom: 15px;
   }
+  .options div {
+    margin: 0em 0.5em 1em 0;
```

```suggestion
    margin: 0 0.5em 1em 0;
```

You don't need a unit for `0` in most cases.